.github/workflows/ci.yml

```diff
@@ -54,15 +54,7 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Load cache
-      uses: actions/cache@v2
-      with:
-        path: |
-          ~/.cargo/bin/
-          ~/.cargo/registry/index/
-          ~/.cargo/registry/cache/
-          ~/.cargo/git/db
-          target/
-        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      uses: Swatinem/rust-cache@v1
```

Comment:
This needs to go after the toolchain is installed:

> selecting a toolchain either by action or manual `rustup` calls should happen before the plugin, as it uses the current rustc version as its cache key

From: https://github.com/Swatinem/rust-cache#example-usage